### PR TITLE
[ACR-1250] Add a new feature for INM

### PR DESCRIPTION
**Pull request by suraj-geddam**
### Change Description ###

Added description of new API feature (INM).

### API Changes ###

ACR: ACR-1250

### CODE REVIEW

**Comment by mijin85cho** [comment]
Hello, @suraj-geddam 

The comment of ACR-1250 is long times ago, so the target branch that should be modified is changed.

For now, master branch is handling Tizen 5.5. So the fix for Tizen 5.0 should be applied to master branch- I changed the working branch to master.

Also, how about following the alphabetical order to list up the features?
If we follow the alphabetical order `http://tizen.org/feature/network.inm` will come before `http://tizen.org/feature/network.m...`

File: docs/application/native/guides/device/system.md [hunk 1]
```diff
@@ -468,6 +468,7 @@ The following table lists the network feature keys:
 | `http://tizen.org/feature/network.wifi.direct.display` | `bool` | The platform returns `true` for this key and the `http://tizen.org/feature/network.wifi` key, if the device supports Wi-Fi Direct display feature. |
 | `http://tizen.org/feature/network.wifi.direct.service_discovery` | `bool` | The platform returns `true` for this key and the `http://tizen.org/feature/network.wifi` key, if the device supports Wi-Fi Direct service discovery. |
 | `http://tizen.org/feature/network.wifi.tdls` | `bool` | The platform returns `true` for this key, if the device supports the Wi-Fi Tunneled Direct Link Setup (TDLS). |
+| `http://tizen.org/feature/network.inm` | `bool` | The platform returns `true` for this key, if the device has supports the Intelligent Network Monitoring (INM) feature. |
```
Comment (rishu-samsung) [inline docs/application/native/guides/device/system.md:471]: @suraj-geddam:

Kindly remove ‘has’ in this sentence, for consistency with other sentences on the page:

The platform returns `true` for this key, if the device supports the Intelligent Network Monitoring (INM) feature.

**Comment by journey2w** [comment]
@suraj-geddam
I left a comment on https://code.sec.samsung.net/jira/browse/ACR-1250.
Please read it and create a new API guide.

**Comment by mijin85cho** [comment]
@suraj-geddam As [@journey2w commented](https://github.com/Samsung/tizen-docs/pull/1075#issuecomment-610688560),
on https://code.sec.samsung.net/jira/browse/ACR-1250, API guide page is also requested.

```
Please write a new API guide for INM as well as add the new feature.

You can add a new file to https://github.com/Samsung/tizen-docs/pull/1075 or  raise a new PR.

- repo: https://github.com/Samsung/tizen-docs.git

- branch: master

- directory: docs/application/native/guides/connectivity
```

So how do you want to process? Are you going to raise a new PR?
If so, I'll merge this PR and wait your another update.

**Comment by rishu-samsung** [comment]
> @suraj-geddam As [@journey2w commented](https://github.com/Samsung/tizen-docs/pull/1075#issuecomment-610688560),
> on https://code.sec.samsung.net/jira/browse/ACR-1250, API guide page is also requested.
> 
> ```
> Please write a new API guide for INM as well as add the new feature.
> 
> You can add a new file to https://github.com/Samsung/tizen-docs/pull/1075 or  raise a new PR.
> 
> - repo: https://github.com/Samsung/tizen-docs.git
> 
> - branch: master
> 
> - directory: docs/application/native/guides/connectivity
> ```
> 
> So how do you want to process? Are you going to raise a new PR?
> If so, I'll merge this PR and wait your another update.

@suraj-geddam A gentle reminder, kindly update the PR as suggested.

**Comment by suraj-geddam** [comment]
Hi, please merge this PR. Will raise another for the new API guide. Thanks.

**Comment by mijin85cho** [comment]
I tried to merge this PR, but it looks like the suraj-geddam:tizen_5.0_prepare branch is too old compared to current Samsung:master. I can't merge with current state.

@suraj-geddam Would you please rebase your PR?

**Pull request merged**